src/persist/index.ts

```diff
@@ -1,6 +1,6 @@
 export * from './adapter';
 export * from './fileAdapter';
-export * from './stringAdapter';
+export * from './memoryAdapter';
```

Comment:
this will lead to breaking change. Please also export memory adapter.

Reply:
Revert the `stringAdapter`.